README.md

```diff
@@ -1,13 +1,21 @@
 # StackJanitor
 
-[![Maintainability](https://api.codeclimate.com/v1/badges/36f6ce4580dafa42efc1/maintainability)](https://codeclimate.com/github/lendi-au/StackJanitor/maintainability)
+![StackJanitor](https://i.imgur.com/KMkUEo6.jpg)
 
-StackJanitor will monitor deployed development AWS CloudFormation Stacks and eliminate unused stacks based on the events activities to mitigate cloud costs.
+StackJanitor is a tool to clean up AWS CloudFormation Stacks in development environment based on TTL (expiration time) and git pull request events. It cleans up unused development CFN stacks to help you save AWS cloud development costs.
 
 ## How it works
 
+StackJanitor automatically monitors all the CloudFormation stacks that you create in the AWS account. If stackjanitor tag is enabled, StackJanitor will put a default TTL (based on your configurations) for the Stack. It will clean up the CFN Stack when TTL is expired.
+
+**Example:** You have deployed a development CFN Stack with a tag `stackjanitor = "enabled"`. if the default TTL is set as 7 days (604800 seconds) in the `serverless.yml`, the CFN stack will be cleaned up automatically after 7 days. Any `UpdateStack` will refresh the TTL.
+
 ![StackJanitor Architecture](./StackJanitor.png "StackJanitor Architecture")
 
+StackJanitor also works with Github and Bitbucket Pull request events. If your CloudFormation Stack has tag for specific git branch and repo, StackJanitor can clean up the Stack when pull request is merged or declined for the branch.
```

Comment:
```suggestion
StackJanitor can also delete stacks using either a Github or Bitbucket webhook. Our current workflow uses merged pull request events to trigger the webhook so that we can quickly delete stacks as part of our development workflow.

To use this feature, your CloudFormation Stack must also be tagged with `BRANCH` and `REPOSITORY` in order for StackJanitor to find the stack associated with your branch/repository combination and delete it.
```